src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -93,7 +93,7 @@ const WorkspaceTabs = ({
       h(WorkspaceMenu, {
```

Comment:
I think we would want to render it as disabled instead of not rendering at all, but given that we already do the dynamic loading/enabling for the workspaces list, I don't think it is worth making the behavior different here.